remoting/getty/getty_client.go

```diff
@@ -38,9 +38,9 @@ import (
 )
 
 var (
-	errSessionNotExist   = perrors.New("session not exist")
-	errClientClosed      = perrors.New("client closed")
-	errClientReadTimeout = perrors.New("client read timeout")
+	errSessionNotExist               = perrors.New("session not exist")
+	errClientClosed                  = perrors.New("client closed")
+	errClientReadTimeoutOrDecoedFail = perrors.New("client read timeout or decode pkg fail")
```

Comment:
"client read timeout or decode pkg fail" --> "maybe the client read timeout or fail to decode tcp stream in Writer.Write"